games-roguelike/moria/moria-5.7.10.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake-utils
+
+DESCRIPTION="Rogue-like D&D curses game similar to nethack (BUT BETTER)"
+HOMEPAGE="https://umoria.org/"
+SRC_URI="https://github.com/dungeons-of-moria/umoria/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND=">=sys-libs/ncurses-6:0"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/umoria-${PV}"
+BUILD_DIR="${S}"
```

Comment:
this is useless

Reply:
Unfortunately the game's build instructions order to run cmake from the source top directory. It fails if you run it in `${WORKDIR}/${P}_build`

Reply:
Do this instead.

`CMAKE_IN_SOURCE_BUILD=true`

Reply:
@chewi Thanks!